deployments/bsc.diamond.staging.json

```diff
@@ -41,9 +41,9 @@
         "Name": "LIFuelFacet",
         "Version": "1.0.0"
       },
-      "0x3004db169fa7956609A872736452E4951D4BDA8b": {
-        "Name": "OFTWrapperFacet",
-        "Version": "2.0.0"
```

Comment:
Why does the script replace this other deployed facet, both should be listed in the file.